src/Data/Aeson/Internal/ByteString.hs

```diff
@@ -84,4 +84,4 @@ liftSBS sbs = withBS bs $ \_ len -> [| unsafePackLenLiteral |]
 
 unsafePackLenLiteral :: Int -> Addr# -> ShortByteString
 unsafePackLenLiteral len addr# =
-    accursedUnutterablePerformIO $ createFromPtr (Ptr addr#) len
+    unsafeDupablePerformIO $ createFromPtr (Ptr addr#) len
```

Comment:
It might be worth pointing out that this implementation was taken almost verbatim from `bytestring`. This would serve as a pointer to those interested in why `unsafeDupablePerformIO` in particular was chosen.

Reply:
Done